iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -56,14 +58,19 @@ void CmdLineParser::parse(int argc, char* argv[], const CmdLineArgumentParsingMo
             std::cout << "-l, --log-level <LEVEL>           Set log level." << std::endl;
             std::cout << "                                  <LEVEL> {off, fatal, error, warning, info, debug, verbose}"
                       << std::endl;
-            std::cout << "-c, --compatibility               Set compatibility check level between runtime and RouDi"
+            std::cout << "-c, --compatibility               Set compatibility check level between runtime and RouDi."
                       << std::endl;
             std::cout << "                                  off: no check" << std::endl;
             std::cout << "                                  major: same major version " << std::endl;
             std::cout << "                                  minor: same minor version + major check" << std::endl;
             std::cout << "                                  patch: same patch version + minor check" << std::endl;
             std::cout << "                                  commitId: same commit ID + patch check" << std::endl;
             std::cout << "                                  buildDate: same build date + commId check" << std::endl;
+            std::cout << "-k, --kill-delay <UINT>           Sets the delay when RouDi kills the apps hard, if they"
+                      << std::endl;
+            std::cout << "                                  have't responded after the first soft kill, in seconds."
```

Comment:
how about `graceful termination request` instead of `soft kill`?

Reply:
Being explicit now.